### DISCUSSION TITLE
Is it possible to add the code-server to an allready existing webserver on a raspberry pi 4 4GB

### DISCUSSION THREAD

**Discussion by Playit3110** [coder/code-server#2068] [Q&A] [answered]
I tried to download and run the install.sh file, I also tried with the .deb file to install code-server, but none of them worked. Is it possible to compile code-server on a raspberry pi? And if yes, how?

**Comment by bpmct** [comment]
Yes! Follow these instructions for installing on a Raspberry Pi: https://github.com/cdr/code-server/issues/35#issuecomment-623241264

This will be more supported in the future.

**Comment by bpmct** [comment, marked as answer]
As for installing on an existing webserver, you can use a reverse proxy on your web server to proxy the code-server port on a certain port. The configuration for that depends on what web server you're using.

**Reply by Playit3110** [reply]
Thank you
